docs/05_operations/02_commands/04_Genx_CLI.md

```diff
@@ -0,0 +1,450 @@
+```
+
+This will use the latest available version of the CLI and create a new application called **myApp** usind the default seed **blank-app-seed**.
+
+### Local
+
+Once you have a project, you can add Genx CLI as a local dependency to your **package.json**:
+
+``` javascript
+"devDependencies": {
+    ...
+    "@genesislcap/genx": "...",
+    ...
+},
+```
+
+You can find out the latest available version by running `npm info @genesislcap/genx` at any point.
+
+Once you have bootstrapped NPM dependencies, you will be able to call Genx CLI in **package.json** scripts section like so:
+
+``` javascript
+"scripts":{
+    ...
+    "build": "genx <command> [args]"
+    ...
+}
+```
+
+For example:
+
+``` javascript
+"scripts":{
+    ...
+    "build": "genx build -e ENABLE_SSO",
+    "dev": "genx dev -e API_HOST,ENABLE_SSO"
+    "serve": "genx serve",
+    "test": "genx test"
+    ...
+}
+```
+
+now if you execute `npm run build` or `npm run dev`, it will launch Genx CLI installed in your project locally.
+
+## Init
+
+The `init` command creates a new project in the selected directory. The basic structure of this command is:
+
+```terminal
+npx -y @genesislcap/genx@latest init <app_Name>
+```
+
+The `init` command creates a new project called `<app_Name>`. When the command runs, the user will be prompted to answer a set of questions to cover the basic configuration of the project.
+
+This command can be used with the following parameters:
+
+|Parameter | Argument  | Description |
+|----------|-----------|-------------|
+|`-s` or `--seed`|`<location>` | provide a seed where the project will be created |
+| `--ref`| [ref] | provide a branch, tag or commit for creating the project |
+|`-x` or `--skip-optional-prompts` | | omits the prompts and applies default values |
+|`--insecure` | omits SSL certificate verification |
+|`--remote` | | remote seed only; any local seed is ignored |
+|`-l` or `--log-level` |**info** or **verbose** | info (default) or verbose |
+|`-h` or `--help` | | display help information |
+
+### --seed
+
+To use the `-s` or `--seed` parameter to specify the local seed directory, a remote seed or one of the pre-defined seeds provided by genesis.
+
+The pre-defined seeds available are:
+
+- blank-app-seed: a seed with a basic structure of a genesis project.
+- positions-app-seed: a seed with a positions app created.
+- devtraining-seed: a seed with the starting dev training project.
+- servertraining-seed: a seed with the starting server training project.
+- webtraining-seed: a seed with the starting web training project.
+
+Here is an example of creating a new project named **myApp** based on the pre-defined positions-app-seed:
+
+```terminal
+npx -y @genesislcap/genx@latest init myApp -s positions-app-seed
+```
+
+You can use a local seed, to do that follow the example below:
+
+```terminal
+npx -y @genesislcap/genx@latest init myApp -s ./path/to/local-seed
+```
+
+Or you can use a custom remote sed for your app
```

Comment:
Typo. 

Example is for local seed - you can use `-s githubuser/repo` or similar instead